career-growth.md

```diff
@@ -0,0 +1,69 @@
+# Conscious Career Growth
+
+It used to be the case that people would stay with one employer for their entire careers. This is no longer the case today. It is normal and expected for most people to move from role to role throughout their lives. That said, you are at Packback right now for a reason! Your job with Packback is an alliance that benefits both parties:
+
+* You apply your unique skills, vision and expertise to move Packback forward as a company.
+* Your work at Packback pushes you to learn new skills, develop your vision and build new expertise.
+
+This exercise is designed to help you think through your ideal future. We'll be looking at the intersection of how your skills, vision and expertise move Packback forward, and how your growth in turn will help Packback to accelerate towards its mission.
+
+## How This Works
+
+First, this is **not a performance conversation, there is no way to fail at this**. We already have tools in place to help you get to your next promotion (performance reviews every trimester, peer reviews yearly, a regular cadence of opportunities for promotion, (1-on-1s)[1-on-1s.md] for weekly coaching). This exercise is not about your next promotion. Conscious career growth is about longer timescales, so try and forget the next year for now.
+
+**This is not an exercise with an end date**. There is no "up and out." When you reach one career milestone it is simply time to look forward to the next. You'll be exploring the overlap between your aspirations and your career at Packback, and revisiting and tweaking regularly. Going into this meeting, both parties should be in a **directional, aspirational mindset!** Don't be afraid or embarrassed to think big. 
+
+This exercise is done with two meetings:
+
+* A skip-level meeting with your manager's manager where you fill out the three questions below
+* A meeting with your manager as part of a standard 1:1, where you tell them what was discussed and talk through how this fits into your day-to-day role.
+
+We'll revisit this exercise yearly, and you should expect to check in with your manager during performance reviews simply to pulse-check. Have answers ready for the first two questions before the first meeting. You may be confident, unsure, excited or nervous. It doesn't matter, just have answers that make you feel something.
```

Comment:
I think the language about how the employee might feel in this situation is unnecessary. We've already made it clear they're meant to approach this exercise feeling however they feel so this feels just a tad patronizing. I would emphasize how they only need to have the answers for the first two questions and that those answers can be effectively whatever they want.

Reply:
Hmm... I'm still noodling on this one; trying to figure out what to say.

Reply:
I'm moving this to the paragraph above about being in a directional, aspiration mindset. That seems to be where we address the team member's emotional relationship to these conversations.